docs/modules/ROOT/pages/how-to/vshn-example/keycloak-brokering.adoc

```diff
@@ -91,20 +95,34 @@ Logout URL = https://{foreign-host}/auth/realms/{foreign-realm}/protocol/openid-
 User Info URL = https://{foreign-host}/auth/realms/{foreign-realm}/protocol/openid-connect/userinfo
 Client Authentication = "Client secret sent as post"
 Client ID = {foreign-client-id}
-Client Secret = <The secret displayed in {appuio-keycloak} Credentials tab>
+Client Secret = <The secret displayed in {foreign-host} Credentials tab>
 ----
 
-. Add a group mapper to the client via the "Mappers" tab
+. Add a "Claim to Group" mapper to the client via the "Mappers" tab
 +
 [source,subs="attributes+"]
 ----
 Name = import-groups
 Sync Mode Override = inherit
-Mapper Type = Claim to Group Mapper
+Mapper Type = Claim to Group
 Claim = groups
 Contains text = <1>
 Create groups if not exists = True
 ----
 <1> Enter the group name if you only want to sync a certain group
+
+. Add a "Claim to Attribute" mapper to the identity broker via the "Mappers" tab
 +
-NOTE: This step requires that https://github.com/appuio/appuio-keycloak-extensions[APPUiO Keycloak extensions] is installed.
+[source,subs="attributes+"]
+----
+Name = default-organization
+Sync Mode Override = inherit
+Mapper Type = Claim to Attribute
+Claim = groups
+Target attribute = appuio.io/default-organization
+Overwrite existing attribute value = false
+Ignore entries pattern = [ ^Service.* , ^VSHN\s(?!employee).* ]
+Lowercase names = True
+Trim whitespaces = True
+Trim prefix = (Cust|VSHN)\s
```

Comment:
Note: This config causes the default organization for VSHNeers to be `employee`, since the prefix is removed.
We could maybe configure this to `vshn-employee` or use the openshiftroot group (`vshn-openshiftroot`).
This setting should then be in sync with the group mapper, which doesn't yet support the same formatting yet.

Thoughts, opinions?